app/services/calculators/subject_matter_of_dispute_disregard_calculator.rb

```diff
@@ -0,0 +1,29 @@
+module Calculators
+  class SubjectMatterOfDisputeDisregardCalculator < BaseWorkflowService
+    def value
+      total_disputed_asset_value = disputed_capital_value +
+        disputed_property_value +
+        disputed_vehicle_value
+
+      [total_disputed_asset_value, threshold].min
+    end
+
+  private
+
+    def threshold
+      @threshold ||= Threshold.value_for(:subject_matter_of_dispute_disregard, at: submission_date) || 0
```

Comment:
IMHO we shouldn't default this to 0. If this ever happens, I would quite like a crash as the world has just ended (time has gone backwards)

Reply:
@starswan I don't entirely agree on this one. CFE has the functionality to retrospectively assess applications submitted in the past, which is why we have threshold data stretching back several years, and a client-defined submission date. My thought was that to be consistent (and avoid reconstructing what the SMOD upper limit was at various points in recent history), we should only enable SMOD considerations for the period for which we know what the threshold is. But a submission date reasonably far in the past is not a sign that the world has ended.